docs/administration/high-availability/design/octopus-for-high-availability-on-aws.md

```diff
@@ -0,0 +1,59 @@
+
+From the Octopus perspective, how the database is made highly available is really up to you; to Octopus, it's just a connection string. We are not experts on SQL Server high availability, so if you have an on-site DBA team, we recommend using them. There are many [options for high availability with SQL Server](https://msdn.microsoft.com/en-us/library/ms190202.aspx), and [Brent Ozar also has a fantastic set of resources on SQL Server Failover Clustering](http://www.brentozar.com/sql/sql-server-failover-cluster/) if you are looking for an introduction and practical guide to setting it up.
+
+### Shared storage
+
+You will need shared storage that all Octopus nodes can access as Octopus stores a number of files that are not suitable for the database.
+
+To build a highly available Octopus server install, you’re going to need some durable Windows file storage, and in the past, AWS didn’t offer any cloud services that accommodated this. Your first option would be to spin up some more EC2 instances and build your own DFS(Distributed File System).
+
+AWS recently introduced `Amazon FSx`, It’s a native Windows file system built on Windows Server. It includes full support for the SMB protocol, Windows NTFS, and Microsoft Active Directory (AD) integration and is an ideal choice for connecting to your EC2 instances hosting Octopus to store all your Octopus packages and log files.
+
+If you choose to go with Amazon FSx there are some resources that will help you get started:
+- AWS have a [starter guide](https://docs.aws.amazon.com/fsx/latest/WindowsGuide/getting-started.html) which explains how to configure Amazon FSx and connect it up to an EC2 machine.
+- We have an [AWS FSx High Availability blog post](https://octopus.com/blog/aws-fsx-ha) which is a step-by-step guide to connecting up Amazon FSx to your Octopus High Availability Server nodes on Windows.
+
+### Load Balancing in AWS
+
+To distibute traffic to the Octopus web portal on multiple nodes then you will need to use a HTTP loadbalancer. AWS provides a solution to distribute HTTP/HTTPS traffic to EC2 instances, Elastic Load Balancing is a highly available, secure, and elastic load balancer. There are three implementations of ELB;
+
+* [Application Load Lalancer](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/introduction.html)
+* [Network Load Balancer](https://docs.aws.amazon.com/elasticloadbalancing/latest/network/introduction.html)
+* [Classic Load Balancer](https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/introduction.html)
+
+If you are using [Listening Tentacles](/docs/infrastructure/deployment-targets/windows-targets/tentacle-communication.md#polling-tentacles) then we recommend using the Application Load Balancer.
```

Comment:
The link text refers to listening tentacles, but the actual link goes to polling 🐙